test/test_rhel.py

```diff
@@ -46,3 +46,27 @@ def test_build_can_build_tar(build_container):
         "tar",
         "--distro", "centos-10",
     ])
+
+
+@pytest.mark.skipif(os.getuid() != 0, reason="needs root")
+@pytest.mark.skipif(not os.getenv("RHEL_REGISTRY_USER"), reason="needs registry secrets")
+def test_rhel_bootc_anaconda_iso(tmp_path):
+    rhel_registry_user = os.getenv("RHEL_REGISTRY_USER")
+    rhel_registry_secret = os.getenv("RHEL_REGISTRY_TOKEN")
+    bootc_ref = "registry.redhat.io/rhel10/rhel-bootc:latest"
+    subprocess.check_call([
+        "podman",
+        "login", "registry.redhat.io",
+        "--username", rhel_registry_user,
+        # XXX: will briefly appear in process list
+        "--password", rhel_registry_secret,
```

Comment:
Podman supports `--password-stdin` just saying. This is good enough.

Reply:
Thanks for not letting me get away with this ;) I fixed this now :)